### DISCUSSION TITLE
Webhook with condition [Jun 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2323] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/2209  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040> do you have the same behaviour?  
#  
> 🙋‍♂️ **alex.pro**  
#### I didn't want to create a ticket without confirmation here  
#  
> 💁 **o1lab**  
#### please provide your project info  
#  
> 💁 **o1lab**  
#### in the github issue  
#  
> 🙋‍♂️ **alex.pro**  
#### anything else I can help you with?  
#  
> 🙋‍♂️ **alex_zhitmarev**  
#### Got exactly the same behaviour of "on condition" is turned on  
#  
> 🙋‍♂️ **alex.pro**  
#### what's your project info?  
#  
> 🙋‍♂️ **alex_zhitmarev**  
#### sorry. I am new with nocodb - what is project info?  
#  
> 🙋‍♂️ **alex.pro**  
#### Top right corner -> click 3 dots -> copy project info   
#### and paste it here  
#  
> 🙋‍♂️ **alex.pro**  
#### Or you can add comment to GH issue with your project info, it may be useful for developers  
#  
> 🙋‍♂️ **alex_zhitmarev**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **sqlite3**   
#### ProjectOnRootDB: **true**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.91.1**  
#  
> 🙋‍♂️ **alex_zhitmarev**  
#### here it is))  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040> https://www.berrycast.com/conversations/1bca497d-7f6f-5857-aad2-261d74e6818d webhooks sometimes don't work (all webhooks from video worked... I can't say when they stopped working, I see the last successful call was 23 of May)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981183277358522368)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
